delta-app/src/main/java/io/cdap/delta/app/DeltaWorker.java

```diff
@@ -316,4 +367,32 @@ private void startFromLastCommit() throws Exception {
     eventReader.start(offset);
     eventConsumer.start();
   }
+
+  @VisibleForTesting
+  static Map<Integer, Set<TableId>> assignTables(DeltaConfig config) {
+    ParallelismConfig parallelism = config.getParallelism();
+    List<InstanceConfig> instances = parallelism.getInstances();
+    Map<Integer, Set<TableId>> assignments = new HashMap<>();
+    if (!instances.isEmpty()) {
+      int instanceNum = 0;
+      for (InstanceConfig instanceConfig : instances) {
+        assignments.put(instanceNum, instanceConfig.getTables());
+        instanceNum++;
+      }
+      return assignments;
```

Comment:
So here if `numInstances` > `instances.size()`, the instance number will be `instances.size()`. Should we log something since in UI user will see the number of instances is smaller than they specified.